ui/src/routes/app/[id]/+page.svelte

```diff
@@ -75,6 +76,14 @@
 						>
 							Grid View
 						</button>
+
+						{#if viewMode === 'table'}
+							<Switch
+								bind:checked={showImage.v}
+								name="show-image-in-table-toggle"
+								label="Show images"
+							/>
```

Comment:
Following on from my comment(s) from elsewhere, this should just be the naked label & [(Skeleton)](https://www.skeleton.dev/docs/components/switch/svelte) switch.